plugins/polio/js/src/domains/Campaigns/MainDialog/usePolioDialogTabs.tsx

```diff
@@ -50,16 +53,16 @@ export const usePolioDialogTabs = (
                 key: 'riskAssessment',
             },
             {
-                title: formatMessage(MESSAGES.scope),
-                form: ScopeForm,
+                title: formatMessage(MESSAGES.evaluation),
+                form: EvaluationsForms,
                 disabled:
                     !formik.values.initial_org_unit ||
                     formik.values.rounds.length === 0,
                 hasTabError: compareArraysValues(
                     scopeFormFields,
```

Comment:
why the scopeFormFields? it seems to me there should be an `evaluationFormField` where we check all the fields that have been moved out of the old `Rounds` tab. Otherwise the Evaluation tab will never have a correct error state